ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -32,6 +40,24 @@
             @onKeyUp={{this.updateWarningValidation}}
           />
         {{/each}}
+      {{else}}
+        {{! EDIT }}
+        {{#if (eq group "Credentials")}}
+          {{#each fields as |attr|}}
+            <EnableInput data-test-enable-field={{attr.name}} class="field" @attr={{attr}}>
```

Comment:
I thought EnableInput was to be removed? If so, I think the component can be deleted since this was the only place it was used.

Reply:
I ended up keeping it because it handles the masking for the json credentials on [gcp-sm](https://github.com/hashicorp/vault/blob/main/ui/app/models/sync/destinations/gcp-sm.js#L40). I also didn't want to risk any kind of regression in this area, so the double "confirm edit" and then eye icon with masked input seemed acceptable. I went back and forth on this and opted for a more conservative of the approaches.

Reply:
Ah cool! Thanks for explaining